src/fides/api/service/connectors/query_configs/bigquery_query_config.py

```diff
@@ -227,3 +227,45 @@ def format_fields_for_query(
             else:
                 formatted_fields.append(field_path.levels[0])
         return formatted_fields
+
+    def generate_raw_query(
+        self, field_list: List[str], filters: Dict[str, List[Any]]
+    ) -> Optional[TextClause]:
+        """
+        Allows executing a somewhat raw query where the field_list and filters do not depend
+        on the Node or Graph structure.
+
+        This is an override of the base class method that supports nested fields for BigQuery.
```

Comment:
How would this function handle `JSON` types (which are fairly similar to `STRUCT`)?

Or similarly, `ARRAY` types 👍 

I assume it'd fail at runtime, so it'd be ideal to have some guardrails in place to error gracefully.

Reply:
I'll keep this in mind for the main ticket that I have for next sprint